### ISSUE TITLE
prompt an error when drop an element from one list to another [knockout-kendoSortable]

### ISSUE THREAD

**Issue by aufula** [kendo-labs/knockout-kendo#226]
reproduce this issue on page http://kendo-labs.github.io/knockout-kendo/web/Sortable.html
check tab "Sorting between lists"

**Comment by aufula** [comment]
before:
```javascript
//we are moving the item ourselves via the observableArray, cancel the draggable and hide the animation
e.sender._cancel();
```
after:
```javascript
 //we are moving the item ourselves via the observableArray, cancel the draggable and hide the animation
e.sender.placeholder.remove();
```

**Comment by bmsmg** [comment]
I have also run into this issue.  As you stated, it is reproducible from the knockout-kendo demo pages.  Your fix does the trick, though I'm unsure why the draggedElement becomes null prior to calling _cancel.  Thanks for the workaround.

**Comment by rniemeyer** [comment]
Thanks! Fixed in v0.9.7
